packages/frontend/src/App.tsx

```diff
@@ -226,7 +226,10 @@ const App = () => (
                                                     >
                                                         <Projects />
                                                     </Route>
-                                                    <Route path="/home" exact>
+                                                    <Route
+                                                        path="/home/:projectUuid?"
```

Comment:
Should we not change line #237 in this file too ?

Reply:
I don't think there are any other references to just "/home"

Reply:
I'm now defaulting to projects page and letting it figure out what to show or redirect